### ISSUE TITLE
do not get single service on one host?

### ISSUE THREAD

**Issue by xiyoulaoyuanjia** [zorkian/nagios-api#37]
hi: http_server do not support for getting single service? 
but the comment at the http_service as below.

'''
Get a view of a single service on one host.
 Shows all of the variables for the host object.
'''

**Comment by ghostsquad** [comment]
yes, I'm curious about the same thing. Can you add a comment for URL_REGEX?
https://github.com/zorkian/nagios-api/blob/master/nagios-api#L41

Is this project dead?

**Comment by zorkian** [comment]
I wouldn't say it's dead, I just don't use it in my day to day anymore so I haven't been doing any development. (At Dropbox we have ~100 Nagios shards with enough data that this project doesn't work in any reasonable speed, so we've started using the Thruk frontend + our own custom inhouse monitoring system.)

So, I definitely read comments and PRs, usually. Sometimes I miss them, but I'm happy to be poked and to respond. :)

**Comment by ghostsquad** [comment]
Oh cool. Thruk offers an rest interface as well as a modern web interface?
